materials/module-1/worksheets/generate-pdf

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+rm -rf output
+mkdir output
+
+# Requires: npm install -g markdown-pdf
+markdown-pdf 2-peer-to-peer-file-sharing-on-a-mesh.md -o output/2-peer-to-peer-file-sharing-on-a-mesh.pdf
```

Comment:
Can we make this repurposable to any .md in a folder?

Reply:
Will make one builder for all assets, including the presentation gitbooks, then publish one big Github Release zip